cmd/xdpcap/flags.go

```diff
@@ -194,8 +194,8 @@ func parseFlags(name string, args []string) (flags, error) {
 		FlagSet: fs,
 	}
 
-	flags.UintVar(&flags.filterOpts.perfPerCPUBuffer, "buffer", 8192, "Per CPU perf buffer size to create (`bytes`)")
-	flags.UintVar(&flags.filterOpts.perfWatermark, "watermark", 1, "Perf watermark (`bytes`)")
+	flags.IntVar(&flags.filterOpts.perfPerCPUBuffer, "buffer", 8192, "Per CPU perf buffer size to create (`bytes`)")
```

Comment:
Why `int`? Is there a use case for setting these to negative values?

Reply:
No, it just works better with the Go default to use int for lengths and such.

Reply:
Shrug, I guess that's ok.